code/modules/research/xenobiology/vatgrowing/microscope.dm

```diff
@@ -36,12 +43,16 @@
 	for(var/organism in current_dish.sample.micro_organisms) //All the microorganisms in the dish
 		if(istype(organism, /datum/micro_organism/cell_line))
 			var/datum/micro_organism/cell_line/cell_line = organism
+			var/atom/resulting_atom = cell_line.resulting_atom
+			var/atom_icon = resulting_atom ? sanitize_css_class_name("[initial(resulting_atom.icon)][initial(resulting_atom.icon_state)]") : ""
```

Comment:
Also feel free to correct me if I'm missing something but it seems like 99% of this should be in `ui_static_data`, none of this actively updates much

Reply:
You could swap the petri dish while UI open.

Reply:
We can `update_static_data_for_all_viewers` when someone swaps it yeah? Or will it not work

Reply:
Didn't know about this proc